src/psyclone/tests/psyir/transformations/loop_fusion_test.py

```diff
@@ -67,9 +67,9 @@ def test_fusetrans_error_incomplete():
     assert ("Error in LoopFuseTrans transformation. The target loop must have "
             "four children but found: []" in str(err.value))
 
-    loop1.addchild(Literal("start", INTEGER_TYPE))
-    loop1.addchild(Literal("stop", INTEGER_TYPE))
-    loop1.addchild(Literal("step", INTEGER_TYPE))
+    loop1.addchild(Literal("1", INTEGER_TYPE))
```

Comment:
Please update the copyright date on this file.

Reply:
Done